timescale-forge/vpc-peering.md

```diff
@@ -0,0 +1,141 @@
+
+<highlight type="tip">
+Though VPC peering is still covered by the Timescale Forge trial period, you will be required
+to enter a valid payment method in order to create a VPC.
+</highlight>
+
+### Create a peering connection
+When you have created a Timescale Forge VPC, you are ready to create a peering connection
+between your Forge VPC and your cloud VPC. To do this, expand the dropdown menu by clicking
+the `VPC Peering` column on your Forge VPC.
+
+<img class="main-content__illustration" src="create-peering-connection.png" alt="Expand the VPC Peering dropdown menu and enter info"/>
+
+From this menu, enter the AWS Account ID, AWS VPC ID and AWS region of your
+cloud VPC for the new peering connection. When you have entered the correct
+information, click `Add peering connection` to begin the peering process.
+
+This process is asynchronous, and results in a peering connection sent to your
+AWS account for you to accept. This is an important safety mechanism, never
+accept a peering connection from an unknown account. All Timescale Forge
+peering connections are sent from the AWS account `142548018081`.
+
+<highlight type="tip">
+Make note of the peering connection ID (starting with `pcx-`) as it is used in the next step.
+</highlight>
+
+<img class="main-content__illustration" src="aws-accept-peering-connection.png" alt="Accept peering connection in AWS console after verifying requestor account number"/>
+
+### Network Route & Security
+Once you have accepted the peering connection from the last step, the two VPCs will now
+be peered; however, in order to use this peering connection, you will need to update your
+VPC's route table to include the CIDR block of your peered Forge VPC, and lastly you will
+need to update your VPC's security groups.
+
+#### Route Table
+Within the AWS console, navigate to the
+[Route Tables](https://console.aws.amazon.com/vpc/home#RouteTables:sort=routeTableId)
+dashboard. Select the route table corresponding to your VPC. From the detail menu, select
+the `Routes` tab and click the `Edit routes` button.
+
+<img class="main-content__illustration" src="aws-route-table-routes.png" alt="The AWS Route Tables dashboard with Routes tab expanded"/>
+
+From this view, click `Add route`. In the `Destination` column of the new row,
+enter the CIDR block of the Forge VPC for which the peering connection was
+configured in the previous few steps.
+
+Then in `Target` column, enter the peering connection ID (starting with `pcx-`)
+noted in the previous step where we created the peering connection.
+
+No other configuration is needed here, so you may now click `Save routes`. This
+configuration allows network traffic to flow from your VPC, across the peering
+connection, and over to the Forge VPC where your Timescale services reside.
+
+<img class="main-content__illustration" src="aws-edit-routes.png" alt="Adding a new route table entry for our peering connection"/>
+
+#### Security Groups
```

Comment:
I believe our docs support "steps" like 1, 2, 3, 4.  Maybe sections like that should be shown like this?  Easier to follow & visually nicer.

Reply:
Nice! I'll give that a whirl to see if that helps the flow.

Reply:
I'm going to have to go through and redo the structure of all these, so don't worry too much right now. Do whatever you like the look of, and when I have a template set up, we can do over.